lib/merchant.rb

```diff
@@ -5,15 +5,13 @@ class Merchant
   attr_reader :id,
               :name,
               :created_at,
-              :updated_at,
-              :parent
+              :updated_at
 
-  def initialize(data, parent = nil)
+  def initialize(data)
     @id         = data[:id]
     @name       = data[:name]
     @created_at = data[:created_at]
     @updated_at = data[:updated_at]
-    @parent     = parent
```

Comment:
We may need parent in there for the next iteration...

Reply:
I give you permission to merge this when you feel ready. Just review some of the comments before you do.